shared/bundle_analysis/parser.py

```diff
@@ -174,7 +192,10 @@ def _create_associations(self):
         # associate chunks to assets
         inserts = []
         for chunk_id, asset_names in self.chunk_asset_names_index.items():
```

Comment:
Only build associations of modules/chunks within the same session, when other bundle reports have the same ID/name it would ignore it as each bundle file is independent with one another